packages/graphql-serve/src/components/serveHandler.ts

```diff
@@ -1,7 +1,8 @@
 import { buildGraphbackServer, GraphbackServer } from "../GraphbackServer";
+import { ConflictResolutionStrategyName } from "../runtime";
 
-export const serveHandler = async (argv: { model?: string, port?: number, datasync }): Promise<GraphbackServer> => {
-  const server = await buildGraphbackServer(argv.model, !!argv.datasync);
+export const serveHandler = async (argv: { model?: string, port?: number, datasync: boolean, conflict?: ConflictResolutionStrategyName, deltaTTL?: number }): Promise<GraphbackServer> => {
+  const server = await buildGraphbackServer(argv.model, { datasync: !!argv.datasync, conflict: argv.conflict, deltaTTL: argv.deltaTTL});
```

Comment:
It would be neater to create an interface before passing the argument:

```suggestion
  const server = await buildGraphbackServer(argv.model, dataSyncServeConfig);
```

Easier to read, easier to modify, easier to scale.

Reply:
Thanks for the review!